spec/adapters/signal_adapter/outbound_spec.rb

```diff
@@ -77,8 +78,13 @@
       end
 
       describe 'on error' do
-        before { stub_request(:post, 'http://signal:8080/v2/send').to_return(status: 400) }
-        it { should raise_error(Net::HTTPServerException) }
+        before(:each) { stub_request(:post, 'http://signal:8080/v2/send').to_return(status: 400) }
+
+        it 'ErrorNotifier' do
+          expect(Sentry).to receive(:capture_exception).with(Net::HTTPServerException)
```

Comment:
It would be nice to replace the stubs of `Sentry`, and stub `ErrorNotifier` instead, as we now have that abstraction in place (I like that!). (That way, should we ever move away from Sentry, we can simply adjust the `ErrorNotifier` implementation without having to adjust all the tests. Doesn’t need to be done before merging though.